packages/commerce/src/api/endpoints/cart.ts

```diff
@@ -1,60 +1,62 @@
-import type { CartSchema } from '../../types/cart'
+import type { CartSchema } from '@vercel/commerce/types/cart'
```

Comment:
I know this works but can we keep these imports as relative 🙏 

Ideally avoid imports to the package from the same package for now.

Reply:
Yes, this was not intended. This happened when I replaced the types path in all providers to import just from commerce, so I replaced it with find & replace, and matched this one too unfortunately.